templates/prometheus.yaml

```diff
@@ -16,7 +16,7 @@ spec:
           - name: kubeaddons-service-monitor
```

Comment:
Change this to `kubeaddons-service-monitor-metrics`

Reply:
done